src/modules/Security.rb

```diff
@@ -568,9 +560,8 @@ def write_shadow_config
     # Set SELinux settings
     #
     # @return true on success
-    def write_selinux
-      selinux.mode = @Settings["SELINUX_MODE"]
-      selinux.save
+    def write_lsm_config
```

Comment:
NP: Please, update the method documentation too. It is not "Set SELinux settings" anymore. Maybe somenthing like `Writes the Linux Security Module configuration` makes more sense.